feature-runner/run-features.ts

```diff
@@ -89,33 +89,13 @@ const runner = await runFolder({
 	name: 'hello.nrfcloud.com backend',
 	logObserver: {
 		onDebug: (info, ...args) =>
-			console.error(
-				ts(),
-				chalk.magenta.dim(info.context.keyword),
-				chalk.magenta(info.context.title),
-				...args.map((arg) => chalk.cyan(print(arg))),
-			),
+			console.error(ts(), ...args.map((arg) => chalk.cyan(print(arg)))),
```

Comment:
Unrelated color change. Why?

Reply:
I just remove `info.context` because it is not available since BDD markdown v.6

Reply:
Ah, I see. `context` has been renamed to `step`, it now is only the instance of the Step.